sup3r/utilities/utilities.py

```diff
@@ -1011,8 +1011,11 @@ def calc_height(data, raster_index, time_slice=slice(None)):
         del gp
 
     elif all(field in data for field in ('zg', 'orog')):
+        if len(data['orog'].dims) == 3:
+            hgt = data['orog'][(0,) + tuple(raster_index)]
+        else:
+            hgt = data['orog'][tuple(raster_index)]
```

Comment:
These interp utilities are nearing the point where we should move them all into an interp utilities class. Complexity for this method alone is quite high, and the utilities.py file alone is massive

Reply:
Yeah agreed. Moved to new class.